src/main/java/com/azure/cosmos/kafka/connect/CosmosDBConfig.java

```diff
@@ -50,7 +53,15 @@ public class CosmosDBConfig extends AbstractConfig {
     private static final String COSMOS_CONTAINER_TOPIC_MAP_DISPLAY = "Topic-Container map";
     private static final String COSMOS_CONTAINER_TOPIC_MAP_DOC =
             "A comma delimited list of Kafka topics mapped to Cosmos containers.\n"
-                    + "For example: topic1#con1,topic2#con2.";    
+                    + "For example: topic1#con1,topic2#con2.";
+
+    public static final String COSMOS_SINK_BULK_ENABLED_CONF = "connect.cosmos.sink.bulk.enabled";
+    private static final String COSMOS_SINK_BULK_ENABLED_DOC = "Flag to indicate whether Cosmos DB sink bulk mode is enabled. By default it is true.";
```

Comment:
it will -> as it will ingest faster. But we are going to retry 429 forever, so eventually they should succeeded. 

but I am open to keep the current default behavior.

Reply:
Discussed offline, will use bulk as default